holidays/countries/kazakhstan.py

```diff
@@ -13,81 +13,92 @@
 
 from dateutil.relativedelta import relativedelta as rd
 
-from holidays.constants import WEEKEND, JAN, MAR, MAY, JUL, AUG, DEC
+from holidays.constants import JAN, MAR, MAY, JUL, AUG, OCT, DEC
 from holidays.holiday_base import HolidayBase
 from holidays.utils import _islamic_to_gre
 
 
 class Kazakhstan(HolidayBase):
     """
-    https://www.officeholidays.com/countries/kazakhstan/2020
+    1. https://www.officeholidays.com/countries/kazakhstan/2020
+    2. https://egov.kz/cms/en/articles/holidays-calend
+    3. https://en.wikipedia.org/wiki/Public_holidays_in_Kazakhstan
+    4. https://adilet.zan.kz/rus/docs/Z010000267_/history
     """
 
     country = "KZ"
 
     def _populate(self, year):
         super()._populate(year)
 
-        """Populate the holidays for a given year
-
-        All holidays get observed on weekdays if they fall on weekends, but
-        this has not been implemented as yet.
-        """
         # New Year's holiday (2 days)
-        self[date(year, JAN, 1)] = "New Year"
-        self[date(year, JAN, 2)] = "New Year Holiday"
-
-        # Orthodox Christmas
-        if year >= 2007:
-            self[date(year, JAN, 7)] = "Orthodox Christmas"
-
-        # Women's Day
-        self[date(year, MAR, 8)] = "Women's Day"
-
-        # Nauryz Holiday (3 days)
-        if year >= 2009:
-            self[date(year, MAR, 21)] = "Nauryz"
-            self[date(year, MAR, 22)] = "Nauryz Holiday"
-            self[date(year, MAR, 23)] = "Nauryz Holiday"
-
-        # Unity Day
-        self[date(year, MAY, 1)] = "Unity Day"
-
-        # Defender's Day
+        name = "New Year"
+        self[date(year, JAN, 1)] = name
+        self[date(year, JAN, 2)] = name
+
+        # International Women’s Day
```

Comment:
Not sure whether ' (vs ’) is more standard option.

Reply:
Thank you. Replaced with '